File: 012-Operation Choices/OperationChoices.py
def OperationChoices(c, a, b):
    if c == 1:
        return a + b
    elif c == 2:
        return a - b
    elif c == 3:
        return a * b
    elif c == 4:
        return a // b  # Assuming integer division and b != 0
    else:
        return 0  # Default case

c = int(input("Enter operation code (1-4): "))
a = int(input("Enter value of a: "))
b = int(input("Enter value of b: "))
print("Result:", OperationChoices(c, a, b))

File: 012-Operation Choices/OperationChoices.cpp
#include<iostream>
using namespace std;

int OperationChoices(int c, int a, int b){
    if(c == 1){
        return a+b;
    }
    else if(c == 2){
        return a - b;
    }
    else if(c == 3){
        return a*b;
    }
    else if(c == 4){
        return a/b;
    }
    else{
        return 0;
    }
}
int main(){
    int c,a,b;
    cout<<"Enter a Operation Choice(1-4): ";
    cin>>c;

    cout<<"Enter a Values Of a And b: ";
    cin>>a>>b;

    cout<<"Result: "<<OperationChoices(c,a,b)<<endl;

    return 0;
}